CONTRIBUTING.md

```diff
@@ -21,6 +21,12 @@ eventually merge into `master` for a real Production deployment.
 
 Java 11 is a requirement to build the project.
 
+You also need mongod installed. If you are on mac, I used this guide https://zellwk.com/blog/install-mongodb/ which uses homebrew.
```

Comment:
Is this correct? I don't believe I have it installed and the full build ran fine (using fongo I assume)

Reply:
Maybe this is because I didn't actually do any 'real' development and just ran the build.

Which raises a good point.  How do you 'start' the API locally?

Reply:
I'm not too sure to be fair. I think to use the full potential of the api and mimic the live version you need to use mongo. And I've got lots of documentation on how to actually use the front-end/back-end and a breakdown of each, which I can contribute at some point

Reply:
@johnoliver - Can you add your thoughts here when you get a chance?

Reply:
I personally dont use mongo when developing and rely on fongo. If you were to really try to replicate a prod environment locally then you probably would need a mongo. I rely on tests and then staging as a prod like environment to verify before releasing

Reply:
Cool I didn't realise this, might keep it in there but modify to reflect this